src/nxt_h1proto.c

```diff
@@ -654,6 +656,8 @@ nxt_h1p_header_process(nxt_task_t *task, nxt_h1proto_t *h1p,
     r->target.start = h1p->parser.target_start;
     r->target.length = h1p->parser.target_end - h1p->parser.target_start;
 
+    r->quoted_target = h1p->parser.quoted_target;
+
```

Comment:
What does `quoted_target` mean?

Reply:
See my new comment.
It happens during http parsing.